### PR TITLE
[Merged by Bors] - Remove instances of required arguments with default values

### PR DESCRIPTION
**Pull request by jmcph4**
## Issue Addressed

#4488 

## Proposed Changes

 - Remove all instances of the `required` modifier where we have a default value specified for a subcommand

## Additional Info

N/A

### CODE REVIEW

**Review by paulhauner — APPROVED** [review]
LGTM!

File: lcli/src/main.rs [hunk 1]
```diff
@@ -409,15 +405,13 @@ fn main() {
...
                         .help("The gas limit field in the execution payload generated.")
-                        .required(true)
                         .default_value("30000000"),
                 )
                 .arg(
                     Arg::with_name("file")
                         .long("file")
                         .value_name("FILE")
                         .takes_value(true)
-                        .required(true)
```
Comment (jimmygchen) [inline lcli/src/main.rs]: There's no `default_value` for this one - we may want to keep this one or assign a default value here.

Comment (jmcph4) [reply lcli/src/main.rs]: Good catch!

**Review by jimmygchen — APPROVED** [review]
LGTM! 👍

**Comment by jmcph4** [comment]
> lock Permission denied
> 
> Existing reviewers: [click here to make jmcph4 a reviewer](https://app.bors.tech/repositories/26362/add-reviewer/jmcph4)

That confirms it :smile:

**Comment by michaelsproul** [comment]
re-batching because of the simulator failure

bors r-

**Comment by michaelsproul** [comment]
bors r-
bors +

**Pull request closed without merging**